src/components/ReactPannellum.jsx

```diff
@@ -62,7 +63,7 @@ class ReactPannellum extends React.Component {
         firstScene: sceneId,
       },
       scenes: {
-        [sceneId]: {
+        [sceneId]: JSON.parse(JSON.stringify({
```

Comment:
Hi @jurtomasz13 please explain this line?

Reply:
Hi @hoaiduyit, I added this shallow copy so viewers won't share the same scene config. Because even with separate viewers, as of right now, they still share the same hotspots locations and so on.